doc/contact.rst

```diff
@@ -2,5 +2,5 @@ Contact
 =======
 
 Feel free to submit an
-`issue <https://github.com/yaml2sbml-dev/yaml2sbml/issues`_
-or send us an `e-mail <mailto:marta.ra.matos@gmail.com`_.
+`issue <https://github.com/yaml2sbml-dev/yaml2sbml/issues>`_
+or send us an `e-mail <mailto:marta.ra.matos@gmail.com>`_.
```

Comment:
maybe this email should be Jakob's? xD

Reply:
Fine for me. @jvanhoefer ? We could also drop any e-mail, thus forcing people to create issues.

Reply:
I kind of like the email idea and i don't mind if it's my email address there, i'm just not sure i'm the best/fastest person to answer xD.

Reply:
Would be OK with having my email there :), would be fine with Martas. No strong opinion in any direction...

Reply:
Let's just put both :)